### PR TITLE
Adds Keymap for Iris/blucky

### PR DESCRIPTION
**Pull request by Blucky87**
Adding personal keymap for Keebio Iris

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

File: keyboards/keebio/iris/keymaps/blucky/keymap.c [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```
Comment (fauxpark) [inline keyboards/keebio/iris/keymaps/blucky/keymap.c]:
```suggestion
```

This isn't used anywhere.

File: keyboards/keebio/iris/keymaps/blucky/keymap.c [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+#define _QWERTY 0
```
Comment (noroadsleft) [inline keyboards/keebio/iris/keymaps/blucky/keymap.c]: If you replace Lines 5-8 with the following, you don't have to manually define the indexes for every layer.

```c
enum layer_names {
    _QWERTY,
    _LAYER1,
    _LAYER2,
    _LAYER3
};
```

File: keyboards/keebio/iris/keymaps/blucky/keymap.c [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+#define WM_6     SGUI(KC_TAB)
+
+#define LAYER1   TT(_LAYER1)
+#define LAYER2   TT(_LAYER2)
+
+#define CS_U     C(S(KC_UP))
+#define CS_D     C(S(KC_DOWN))
+#define CS_SPC   C(S(KC_SPC))
+
+#define NONE     KC_NO
```
Comment (noroadsleft) [inline keyboards/keebio/iris/keymaps/blucky/keymap.c]: QMK defines an alias for this keycode automatically.

https://github.com/qmk/qmk_firmware/blob/48cc61b18883a482ad8494e005054e64be5cd8bf/quantum/quantum_keycodes.h#L31

File: keyboards/keebio/iris/keymaps/blucky/keymap.c (hunk 1 above)
Comment (Blucky87) [reply keyboards/keebio/iris/keymaps/blucky/keymap.c]: removec

File: keyboards/keebio/iris/keymaps/blucky/keymap.c (hunk 2 above)
Comment (Blucky87) [reply keyboards/keebio/iris/keymaps/blucky/keymap.c]: changed

File: keyboards/keebio/iris/keymaps/blucky/keymap.c (hunk 3 above)
Comment (Blucky87) [reply keyboards/keebio/iris/keymaps/blucky/keymap.c]: I made my own definition because it used less characters than both that already existed

Comment (yanfali) [reply keyboards/keebio/iris/keymaps/blucky/keymap.c]: So the rational behind is _______ and XXXXXXX is that automated tools can find files and change them automatically. That's not really possible with custom defines. We'd appreciate it if you'd change them back place.

Comment (Blucky87) [reply keyboards/keebio/iris/keymaps/blucky/keymap.c]: updated

**Pull request merged**